### PR TITLE
Reassign articles to wpickett

### LINKED ISSUE #32830: Wade: ownership migration 25 topic
Migrate 25 topics from `ms.author: riande` to Wade per Jill.

Coordinate with @tdykstra who's leading the ownership migration.

### LINKED ISSUE #32831: Migrate 2nd batch of topics from `ms.author: riande` to Wade per Jill.
Migrate  topics from `ms.author: riande` to Wade per Jill.

Coordinate with @tdykstra who's leading the ownership migration.

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #32831 
Fixes #32830

Per management request, reassigning topics for ms.author that we to watch over, keep up the health on so they are more distributed between writers.

This brings the topic total for me to watch over to **148** total in aspnetcore.docs as asked.

The original or current author is maintained in the `author: ` meta field so it remains clear who the author is and credited to.

Added topics recommended by Tom to take, the areas generally cover:

- OpenAPI
- SignalR
- Docker
- Minimal API
- gRPC
- cient-side libman and SPA
- Any topic that was still listed as scaddie for ms.author
- 7 diagnostic/ error topics since everyone seem to have some of those but me, and I was at a loss for the last 7 to take. (Tom didn't originally suggest these 7).

This pr is related to #32852

Feel free to suggest different topics.

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/client-side/bundling-and-minification.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/bundling-and-minification.md) | [Bundle and minify static assets in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/bundling-and-minification?branch=pr-en-us-32870) |
| [aspnetcore/client-side/libman/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/libman/index.md) | [Client-side library acquisition in ASP.NET Core with LibMan](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/libman/index?branch=pr-en-us-32870) |
| [aspnetcore/client-side/libman/libman-cli.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/libman/libman-cli.md) | [Use the LibMan CLI with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/libman/libman-cli?branch=pr-en-us-32870) |
| [aspnetcore/client-side/libman/libman-vs.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/libman/libman-vs.md) | [aspnetcore/client-side/libman/libman-vs](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/libman/libman-vs?branch=pr-en-us-32870) |
| [aspnetcore/client-side/spa-services.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/spa-services.md) | [Use JavaScript Services to Create Single Page Applications in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/spa-services?branch=pr-en-us-32870) |
| [aspnetcore/client-side/spa/intro.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/spa/intro.md) | [Overview of Single Page Apps (SPAs) in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/spa/intro?branch=pr-en-us-32870) |
| [aspnetcore/client-side/spa/react.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/client-side/spa/react.md) | [Use React with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/client-side/spa/react?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/asp0000.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/asp0000.md) | [aspnetcore/diagnostics/asp0000](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0000?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/asp0001.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/asp0001.md) | [ASP0001: Authorization middleware is incorrectly configured](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0001?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/asp0010.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/asp0010.md) | ["ASP0010: Do not use UseStartup with WebApplicationBuilder.WebHost"](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0010?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/mvc1001.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/mvc1001.md) | [MVC1001: Filters cannot be applied to page handler methods](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/mvc1001?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/mvc1002.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/mvc1002.md) | [aspnetcore/diagnostics/mvc1002](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/mvc1002?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/mvc1003.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/mvc1003.md) | [MVC1003: Route attributes cannot be applied to page models](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/mvc1003?branch=pr-en-us-32870) |
| [aspnetcore/diagnostics/mvc1006.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/diagnostics/mvc1006.md) | ["MVC1006: Methods containing TagHelpers must be async and return Task"](https://review.learn.microsoft.com/en-us/aspnet/core/diagnostics/mvc1006?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/metapackage-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/metapackage-app.md) | [Microsoft.AspNetCore.App for ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis.md) | [Minimal APIs quick reference](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/handle-errors.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/handle-errors.md) | [aspnetcore/fundamentals/minimal-apis/handle-errors](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/handle-errors?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/middleware.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/middleware.md) | [Middleware in Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/middleware?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/min-api-filters.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/min-api-filters.md) | [Filters in Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/min-api-filters?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/openapi.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/openapi.md) | [OpenAPI support in minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/openapi?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/parameter-binding.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/parameter-binding.md) | [aspnetcore/fundamentals/minimal-apis/parameter-binding](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/parameter-binding?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/route-handlers.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/route-handlers.md) | [Route Handlers in Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/route-handlers?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/test-min-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/test-min-api.md) | [Test Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/test-min-api?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/minimal-apis/webapplication.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/minimal-apis/webapplication.md) | [WebApplication and WebApplicationBuilder in Minimal API apps](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/webapplication?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/portable-object-localization.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/portable-object-localization.md) | [aspnetcore/fundamentals/portable-object-localization](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/portable-object-localization?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/startup.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/startup.md) | [App startup in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/startup?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/target-aspnetcore.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/target-aspnetcore.md) | [[Visual Studio](#tab/visual-studio)](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore?branch=pr-en-us-32870) |
| [aspnetcore/fundamentals/use-http-context.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/fundamentals/use-http-context.md) | [Use HttpContext in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/use-http-context?branch=pr-en-us-32870) |
| [aspnetcore/grpc/aspnetcore.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/grpc/aspnetcore.md) | [gRPC services with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/aspnetcore?branch=pr-en-us-32870) |
| [aspnetcore/grpc/authn-and-authz.md](https://github.com/dotnet/AspNetCore.Docs/blob/ac4ef072dfd6bf7b2d73e15d63eb44541f765cd2/aspnetcore/grpc/authn-and-authz.md) | [Authentication and authorization in gRPC for ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/authn-and-authz?branch=pr-en-us-32870) |

</details>

> [!NOTE]
> This table shows preview links for the 30 files with the most changes. For preview links for other files in this PR, select <strong>OpenPublishing.Build Details</strong> within [checks](https://github.com/dotnet/AspNetCore.Docs/pull/32870/checks).

### CODE REVIEW

**Comment by wadepickett** [comment]
@tdykstra and @Rick-Anderson, need your feedback on the draft.  Feel free to suggest different articles to switch.

File: aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Get started with Microsoft.AspNetCore.OpenApi
 author: captainsafia
 description: Learn how to generate and customize OpenAPI documents in an ASP.NET Core app
-ms.author: safia
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md:5]: Keep safia, I'm the backup

File: aspnetcore/fundamentals/minimal-apis/overview.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Minimal APIs overview
 author: JeremyLikness
 description: An introduction to the fastest and easiest way to create web API endpoints with ASP.NET Core.
-ms.author: jeliknes
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/overview.md:5]: Keep Jermy, we just need to transfer from @Rick-Anderson to @wadepickett

File: aspnetcore/fundamentals/minimal-apis/responses.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Create responses in Minimal API applications
 author: brunolins16
 description: Learn how to create responses for minimal APIs in ASP.NET Core.
-ms.author: brolivei
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/responses.md:5]: ditto

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by tdykstra — APPROVED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by wadepickett** [comment]
@JamesNK, I have been asked to identify an equal number of topics between the writers that I will be assigned to monitor the health for and actively maintain (as much as we can fit and prioritize into every sprint).

Part of this involves keeping the author metadata set to the author (for example keep it set to you for gRPC topics) while setting ms.author to the topic areas the doc team writer is dedicating time to.  

I'm giving you a heads up since you are one of the development team's most active in creating new docs and maintaining them (at superhero levels).  I didn't want you to be surprised, and to make sure you understood you still attributed to and maintained as `author` and why we are changing `ms.author`.

**Pull request merged**